components/eam/src/chemistry/modal_aero/modal_aero_amicphys.F90

```diff
@@ -1504,7 +1505,7 @@ subroutine mam_amicphys_1gridcell(          &
          misc_vars_aa_sub(jsub),                     & 
          Hconc_sav , awater,                          & !MW: to save aerosol pH (dsj+zlu)
          troplev_i) !kzm:for strat. aerosol cal.
-
+      call t_startf ('amicphys_1subarea_clear')
```

Comment:
Another t_startf -> t_stopf change is needed here.

Reply:
thank you for pointing out that.